main.go

```diff
@@ -97,7 +108,8 @@ func callbackHandler(w http.ResponseWriter, r *http.Request) {
 				return nil
 			}
 
-			if event.Source.UserID != "U00d6815afb6f57eb9ac959614d2520db" && event.Source.UserID != "U5b9f3b6e6636930d434372d70ce9c9b0" {
+			_, ok := userList[event.Source.UserID]
+			if ok == false {
```

Comment:
we can combine 111 and 112 to 
`
if _, exist := userList[event.Source.UserID]; if !exist {
        return nil
}
`